lib/shortenator.rb

```diff
@@ -30,14 +30,15 @@ def search_and_shorten_links(
       domains = config.domains,
       ignore_200_check: config.ignore_200_check,
       tags: config.tags,
-      additional_tags: []
+      additional_tags: [],
+      group_guid: config.group_guid
     )
       @all_tags = tags + additional_tags
       validate_config
 
-      client = bitly_client(token: config.bitly_token)
+      client = bitly_client
```

Comment:
Is this removal separate from the rest of the change? (Given it's the default value here anyways)

Reply:
I missed this; I prefer the version that passes in the config.

Reply:
it was something that I realize was unneeded, the method call already looks at the config, so nothing is changed, just removing an unneeded passing of an argument.